src/runtime/components/data/Table.vue

```diff
@@ -59,6 +47,18 @@
             </slot>
           </td>
         </tr>
+
+        <tr v-for="(row, index) in rows" v-else :key="index" :class="[ui.tr.base, isSelected(row) && ui.tr.selected, $attrs.onSelect && ui.tr.active]" @click="() => onSelect(row)">
```

Comment:
You should wrap this in a `<template>` otherwise the `v-else` will apply on each `v-for` iteration.